src/components/common/EntityInfo.vue

```diff
@@ -32,6 +32,7 @@
         </menu>
       </v-col>
 
+      <!-- *** TODO: verify that cont in NR displays here per designs -->
```

Comment:
It does:

![image](https://github.com/user-attachments/assets/86640b00-b893-4d55-b04e-9443330b3f39)